MODULE.bazel

```diff
@@ -6,6 +6,7 @@ module(
 
 bazel_dep(name = "bazel_skylib", version = "1.3.0")
 bazel_dep(name = "platforms", version = "0.0.5")
+bazel_dep(name = "cgrindel_bazel_starlib", version = "0.15.0")
```

Comment:
can this one be a dev dependency too? i know it's used in tools/ but those are really dev tools

Reply:
I didn't mark it as a dev dependency, because we do [advertise the use of the tool to generate asset declarations](https://github.com/keith/buildifier-prebuilt#option-3-quick-easy-and-canonical).

Reply:
yea i guess i expected folks to run that in this repo as opposed to using their version of it, but sounds fine